app/src/pages/Lobby/Inputs/index.tsx

```diff
@@ -25,41 +24,42 @@ export function UsernameInput(props: {
   const [updatePlayer] = useUpdatePlayerMutation()
   const [value, setValue] = React.useState(props.username || "")
   return (
-    <>
-      <Box display="flex" alignItems="center" justifyContent="flex-start">
+    <form
+      onSubmit={event => {
+        event.preventDefault()
+        updatePlayer({
+          variables: {
+            id: props.playerId,
+            input: { username: value }
+          }
+        })
+      }}
+    >
+      <Box display="flex" alignItems="flex-start" justifyContent="flex-start">
         <Box>
           <TextField
             label="Username"
             variant="outlined"
-            size="medium"
+            size="small"
             defaultValue={props.username || ""}
             onChange={({ target: { value } }) => {
               setValue(value)
             }}
+            helperText={<HelperText>Emojis encouraged! 🌍🚀✨</HelperText>}
           />
         </Box>
         <Box pl={2}>
           <Button
+            type="submit"
             disabled={value === ""}
             variant="outlined"
             size="large"
-            onClick={() => {
-              updatePlayer({
-                variables: {
-                  id: props.playerId,
-                  input: { username: value }
-                }
-              })
-            }}
           >
-            Submit
+            Join
           </Button>
         </Box>
       </Box>
-      <Box pt={1} color={grey[600]}>
-        Emojis encouraged! 🌍🚀✨
-      </Box>
-    </>
+    </form>
```

Comment:
This was a random change to the username input UI:

<img width="389" alt="Screen Shot 2020-05-09 at 5 27 40 PM" src="https://user-images.githubusercontent.com/847532/81485355-67699880-921a-11ea-8b8f-9264ad1affba.png">

I mostly wanted to move this to a form so <kbd>ENTER</kbd> works.

Reply:
looks good, and standard - https://stackoverflow.com/questions/58699898/submit-a-form-using-enter-key-with-material-ui-core-button-in-react-js

Reply:
couldn't get the styling to look right on mobile in a trivial way (the form was seemingly expanding the input beyond a friendly width in mobile iphone x) - let's defer to a separate PR!

Reply:
oh actually it was the helper text